aws.yaml

```diff
@@ -295,9 +295,58 @@
       version: 3.10.1
     - name: etcd
       version: 3.3.17
-- version: 9.0.0
+- version: 9.0.1
   state: active
   active: true
+  date: 2020-03-09T19:00:00Z
+  apps:
+    - app: cert-exporter
+      version: 1.2.1
+    - app: chart-operator
+      version: 0.11.5
```

Comment:
Sorry to be a pain but we need to use 0.12.0. I have 2 more PRs to merge then I'll create the release.